ratpack/src/main/resources/User.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS user;
+CREATE TABLE user (
+  id bigint auto_increment primary key,
+  title varchar(255),
+  name varchar(255),
+  country varchar(255)
+);
+
+insert into user values(1,'Mr.','Norman', 'USA');
+insert into user values(2,'Mr.','Lewis', 'USA');
```

Comment:
It's a small point, but:

- `Mr.` would conventionally be just `Mr`
- People usually have two names - maybe choose movie stars
- Shall we have a mix of genders to be inclusive to our female readers?

Reply:
agreed